src/views/index.vue

```diff
@@ -69,11 +70,11 @@ onMounted(async () => {
       >
         <div class="flex items-center gap-2">
           <Avatar image="/logo.png"></Avatar>
-          <IconField v-tooltip="'Unimplemented now'">
+          <IconField v-tooltip>
             <InputIcon>
               <i class="pi pi-search"></i>
             </InputIcon>
-            <InputText size="small" placeholder="Search" disabled />
+            <InputText v-model="searchtext" size="small" placeholder="Search" />
```

Comment:
This feature still not implemented, please do not remove `disabled` label for this.